packages/pytest-simcore/src/pytest_simcore/docker_compose.py

```diff
@@ -118,6 +118,10 @@ def env_file_for_testing(
             file=fh,
         )
         for key, value in sorted(testing_environ_vars.items()):
+            # ensure OpenTelemetry is not enabled
+            if key.startswith("TRACING_") or key.endswith("_TRACING"):

```

Comment:
IMO this is calling for troubles ... 
I would rather specify that explicitly somewhere and not here hidden
Why not simply specify that in the main `conftest.py` of every test?